### PR TITLE
Rollout improved example converter

### PR DESCRIPTION
**Pull request by t0yv0**
Rolls out improved TF example converter.

### CODE REVIEW

File: provider/cmd/pulumi-resource-docker/schema.json [hunk 1]
```diff
@@ -3629,7 +3629,7 @@
...
+            "description": "\u003c!-- Bug: Type and Name are switched --\u003e\nManages the lifecycle of docker image in a registry. You can upload images to a registry (= `docker push`) and also delete them again\n\n",
```
Comment (blampe) [inline provider/cmd/pulumi-resource-docker/schema.json:3632]: @t0yv0 is it expected for some of these examples to be dropped? This seems like a step backward, but maybe they were just malformed or didn't convert cleanly?

**Review by blampe — APPROVED** [review]
Question around whether we expect to lose some examples but otherwise LGTM

File: .ci-mgmt.yaml [hunk 2]
```diff
@@ -34,3 +38,4 @@ actions:
       uses: webfactory/ssh-agent@v0.7.0
       with:
         ssh-private-key: ${{ secrets.PRIVATE_SSH_KEY_FOR_DIGITALOCEAN }}
+pulumiConvert: 1
```
Comment (blampe) [inline .ci-mgmt.yaml:41]: I'm surprised this file even exists -- I'm assuming it takes precedence over https://github.com/pulumi/ci-mgmt/blob/master/provider-ci/providers/docker/config.yaml? Ugh.

Comment (t0yv0) [reply .ci-mgmt.yaml:41]: Yes this file here is the source of truth, and if you run `make ci-mgmt` it resyncs derivative files to it.

**Comment by t0yv0** [comment]
```
jq -r '.|select(.ExampleName=="#/resources/docker:index/registryImage:RegistryImage").OriginalHCL' cov/byExample.json                                                                                                                  ~/bugs/ci-mgmt-889/pulumi-docker
resource "docker_registry_image" "helloworld" {
  name          = docker_image.image.name
  keep_remotely = true
}

resource "docker_image" "image" {
  name = "registry.com/somename:1.0"
  build {
    context = "${path.cwd}/absolutePathToContextFolder"
  }
}
```

```
t0yv0@Antons-MacBook-Pro> jq -r '.|select(.ExampleName=="#/resources/docker:index/registryImage:RegistryImage")' cov/byExample.json                                                                                                                              ~/bugs/ci-mgmt-889/pulumi-docker
{
  "ExampleName": "#/resources/docker:index/registryImage:RegistryImage",
  "OriginalHCL": "resource \"docker_registry_image\" \"helloworld\" {\n  name          = docker_image.image.name\n  keep_remotely = true\n}\n\nresource \"docker_image\" \"image\" {\n  name = \"registry.com/somename:1.0\"\n  build {\n    context = \"${path.cwd}/absolutePathToContextFolder\"\n  }\n}\n",
  "ConversionResults": {
    "csharp": {
      "FailureSeverity": 2,
      "FailureInfo": "Terraform input not yet implemented: path",
      "Program": "",
      "TranslationCount": 1
    },
    "go": {
      "FailureSeverity": 2,
      "FailureInfo": "Terraform input not yet implemented: path",
      "Program": "",
      "TranslationCount": 1
    },
    "java": {
      "FailureSeverity": 2,
      "FailureInfo": "Terraform input not yet implemented: path",
      "Program": "",
      "TranslationCount": 1
    },
    "python": {
      "FailureSeverity": 2,
      "FailureInfo": "Terraform input not yet implemented: path",
      "Program": "",
      "TranslationCount": 1
    },
    "typescript": {
      "FailureSeverity": 2,
      "FailureInfo": "Terraform input not yet implemented: path",
      "Program": "",
      "TranslationCount": 1
    },
    "yaml": {
      "FailureSeverity": 2,
      "FailureInfo": "Terraform input not yet implemented: path",
      "Program": "",
      "TranslationCount": 1
    }
  }
}
```

**Comment by t0yv0** [comment]
Regarding the one example it appears to be another instance of https://github.com/pulumi/pulumi-converter-terraform/issues/141

**Pull request merged**